src/components/PlaintextReader.vue

```diff
@@ -0,0 +1,82 @@
+<!--
+  - @copyright Copyright (c) 2019 Julius Härtl <jus@bitgrid.net>
+  -
+  - @author Julius Härtl <jus@bitgrid.net>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<EditorContent v-if="editor" id="read-only-editor" :editor="editor" />
+</template>
+
+<script>
+import { Editor } from '@tiptap/core'
+import { PlainText } from './../extensions/index.js'
+import { EditorContent } from '@tiptap/vue-2'
+import escapeHtml from 'escape-html'
+
+export default {
+	name: 'PlaintextReader',
+	components: { EditorContent },
+	props: {
+		content: {
+			type: String,
+			required: true,
+		},
+	},
+	data: () => {
+		return {
+			editor: null,
+		}
+	},
+
+	computed: {
+		htmlContent() {
+			return '<pre>' + escapeHtml(this.content) + '</pre>'
+		},
+	},
+
+	watch: {
+		content() {
+			this.updateContent()
+		},
+	},
+
+	mounted() {
+		this.editor = this.createEditor()
```

Comment:
We do not it be reactive.
Try to do not declare it as data and use `this.$editor` instead of `this.$editor`